tensorflow/core/kernels/resource_variable_ops.cc

```diff
@@ -62,15 +62,15 @@ limitations under the License.
 #include "tensorflow/core/framework/op_kernel.h"
 #include "tensorflow/core/framework/register_types.h"
 #include "tensorflow/core/framework/resource_mgr.h"
+#include "tensorflow/core/framework/resource_var.h"
```

Comment:
`tensorflow/core/kernels:resource_variable_ops does not depend on a module exporting 'third_party/tensorflow/core/framework/resource_var.h'`